db/dcrpg/pgblockchain.go

```diff
@@ -3160,6 +3172,32 @@ func (pgb *ChainDB) blockFees(charts *cache.ChartData) (*sql.Rows, func(), error
 	return rows, cancel, nil
 }
 
+// appendAnonymitySet sets or updates a series of per-block privacy participation.
+// This is the Fetcher half of a pair that make up a cache.ChartUpdater. The
+// Appender half is appendPrivacyParticipation.
+func (pgb *ChainDB) privacyParticipation(charts *cache.ChartData) (*sql.Rows, func(), error) {
+	ctx, cancel := context.WithTimeout(pgb.ctx, pgb.queryTimeout)
+
+	rows, err := retrievePrivacyParticipation(ctx, pgb.db, charts)
+	if err != nil {
+		return nil, cancel, fmt.Errorf("chartPrivacyParticipation: %v", pgb.replaceCancelError(err))
+	}
+	return rows, cancel, nil
+}
+
+// anonymitySet sets or updates a series of per-block anonymity set.
```

Comment:
Also wrap this comment please.  I suggest a comment formatter, like the [Rewrap extension](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap) if you happen to be using VSCode.